wpilibc/src/main/native/cpp/DriverStation.cpp

```diff
@@ -28,6 +32,42 @@ struct MatchInfoData {
   int replayNumber = 0;
   DriverStation::MatchType matchType = DriverStation::MatchType::kNone;
 };
+
+class MatchDataSender {
+ public:
+  std::shared_ptr<nt::NetworkTable> table;
+  nt::NetworkTableEntry metaDataKey;
```

Comment:
I'd also name this one specifically `typeMetadata` to leave room for other possible metadata entries in the future